public/locales/ar/translation.json

```diff
@@ -19,6 +19,13 @@
     }
   },
   "home": {
+    "genderViolenceMain": {
+      "headerPartOne": "أنت لست لوحدك",
+      "headerPartTwo": "هنا بجانبك Safe Place",
+      "span": "",
```

Comment:
translations should know nothing about html.

what you want to do here is something in this answer: 
https://stackoverflow.com/questions/16038458/html-tags-in-i18next-translation

you should also generally never do this partone parttwo thing. i think jennet might have worked on something similar , talk to her

Reply:
@louisrli done